### DISCUSSION TITLE
wangeditor v5 无序列表经常失效

### DISCUSSION THREAD

**Discussion by Gerres** [wangeditor-team/wangEditor#4661] [Q&A] [answered]
v5.1.14版本：
列表的样式经常是小，小黑点总是出不来，有时候能出来，但是大部分情况小黑点变成了空白

**Comment by Gerres** [comment]
不光是样式丢了，而且有文字删除时候光标会错位到盒子外边，如何输入不了文字。。。

**Reply by Gerres** [reply]
视频：https://outin-ec35324ae63f11ecaaca00163e1c9256.oss-cn-shanghai.aliyuncs.com/sv/15093d0e-182689e0cdd/15093d0e-182689e0cdd.mp4?Expires=1659616293&OSSAccessKeyId=LTAIrkwb21KyGjJl&Signature=3QO4fzkBBR7GLK%2FpWJiBiokWQj4%3D

**Comment by wangfupeng1988** [comment, marked as answer]
去提交 issue ，写明复现步骤

**Reply by Gerres** [reply]
已解决，原因：其他第三方库造成的全局样式冲突问题
